lib/pymedphys/_dicom/dose.py

```diff
@@ -314,3 +318,72 @@ def require_patient_orientation(ds, patient_orientation):
             "The supplied dataset has a patient "
             f"orientation other than {patient_orientation}."
         )
+
+
+def sum_doses_in_datasets(
+    datasets: Sequence["pydicom.dataset.Dataset"],
+) -> "pydicom.dataset.Dataset":
+    """Sum two or more DICOM dose grids and save to new DICOM RT
+    Dose dataset"
+
+    Parameters
+    ----------
+    datasets : sequence of pydicom.dataset.Dataset
+        A sequence of DICOM RT Dose datasets whose doses are to be
+        summed.
+
+    Returns
+    -------
+    pydicom.dataset.Dataset
+        A new DICOM RT Dose dataset whose dose is the sum of all doses
+        within `datasets`
+    """
+
+    if not len(datasets) >= 2:
+        raise ValueError("At least two datasets must be provided for comparison")
```

Comment:
My preference would be for keeping the API aligned to other sum functions in the Python ecosystem where it allows a "sum" of one element:

```python
In [1]: sum([1])
Out[1]: 1

In [2]: import numpy as np

In [3]: np.sum([1])
Out[3]: 1
```

A benefit I can think of, potentially there is a variable amount of plans in a sequence and the total of all the plans is wanted as a return, even if it is just one plan.

---

To the point where I think it'd be worth having a unit test testing this edge case.